src/euphorie/client/browser/organisation.py

```diff
@@ -385,6 +395,41 @@ def default_target_view(self):
             return f"{view}#org-{organisation.organisation_id}"
         return view
 
+    @property
+    def email_from_name(self):
+        return api.portal.get_registry_record("plone.email_from_name")
+
+    @property
+    def email_from_address(self):
+        return api.portal.get_registry_record("plone.email_from_address")
+
+    def notify_inviter(self):
+        mailhost = api.portal.get_tool("MailHost")
+        invitee = self.webhelpers.get_current_account()
+        invitee_firstname = invitee.first_name or invitee.last_name or invitee.loginname
+        body = self.email_template(
+            inviter=self.inviter_firstname,
+            invitee=invitee_firstname,
+            organisation=self.organisation.title,
+        )
+        subject = api.portal.translate(
+            _(
+                "subject_notify_invite_accept",
+                default="Confirm OiRA membership in '${organisation}'",
+                mapping={"organisation": self.organisation.title},
+            )
+        )
+        mail = CreateEmailTo(
+            self.email_from_name,
+            self.email_from_address,
+            invitee.email,
+            subject,
+            body,
+        )
+
+        mailhost.send(mail, self.inviter.email, self.email_from_address, immediate=True)
```

Comment:
This looks like an ancient way of doing this thing.
Can you convert it to a more modern stack?